.github/workflows/ci-cd.yml

```
@@ -159,7 +153,74 @@ jobs:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata
+        id: meta
+        uses: docker/metadata-action@030e881283bb7a6894de51c315a6bfe6a94e05cf # v6
+        with:
+          images: |
+            ${{ env.GHCR_REGISTRY }}/${{ env.IMAGE_NAME }}
+          flavor: |
+            latest=false
+          tags: |
```

actionlint (1.7.12)

[error] property "release" is not defined in object type {build: {outputs: {released: string; tag: string; version: string}; result: string}}

(expression)